.devcontainer/devcontainer.json

```
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+// If you're wondering how to get SSH working for git pushes, make sure to ssh-add ~/.ssh/<YOURPRIVATEKEY> on your host system
+// to make sure the ssh agent has your key
+{
+	"name": "Go",
```

Nit: could we rename this to something slightly more specific, e.g.:
```suggestion
	"name": "Beats Development Container",
```